.github/workflows/build_on_push_pr.yml

```diff
@@ -0,0 +1,72 @@
+name: Build package on Push or Pull Request
+
+on:
+  push
+  
+jobs:
+  Prepare:
+    name: 'Prepare'
+    uses: Ultimaker/embedded-workflows/.github/workflows/prepare_env.yml@main
+    with:
+      BUILD_DOCKER_CACHE: true
+    secrets: inherit
+
+  Shellcheck:
+    name: 'Test'
+    needs: Prepare
+    uses: Ultimaker/embedded-workflows/.github/workflows/shellcheck.yml@main
+    secrets: inherit
+
+  Flake8:
+    name: 'Test'
+    needs: Prepare
+    uses: Ultimaker/python-quality-control/.github/workflows/flake8.yml@master
+    with:
+      PARENT_BRANCH: 'master/s-line'
```

Comment:
We just changed the default branch from `master/s-line` to `main`, so please update this workflow.